### DISCUSSION TITLE
Add support for OpenMandriva

### DISCUSSION THREAD

**Discussion by berolinux** [microsoft/WSL#6177] [General] [answered]
Currently wsl --install offers a series of distributions, but OpenMandriva https://openmandriva.org/ is not in the list.

I am one of the main developers of OpenMandriva; is there any special download we need to provide to be added? I presume you essentially need a root filesystem tarball, docker image or similar, given WSL has a custom kernel?

**Comment by ghost** [comment, marked as answer]
The distros in `wsl --install` are from the Microsoft Store. Publish your distro in the Microsoft Store

From the docs:

> To submit to the Store, you will need to work with us to receive publishing approval. If you are a Linux distribution owner interested in adding your distribution to the Microsoft Store, please contact wslpartners@microsoft.com.
